components/cards/cards.css

```diff
@@ -0,0 +1,146 @@
+body {
+    background-color: var(--DEE2E6);
+}
+
+
+/* useless CSS prior to this */
+.card-shadow {
+    box-shadow: 0 4px 8px 0 var(--PURE-BLACK), 0 6px 20px 0 var(--PURE-BLACK);
+}
+
+.card-element-horizontal {
+    width: 35rem;
+    background-color: var(--PURE-WHITE);
+    border-radius: 4px;
+}
+
+.card-element-horizontal>br {
+    color: var(--DEE2E6);
+}
+
+/* card element main content */
+.card-element__main {
+    display: flex;
+
+}
+
+.card__image-container {
+    width: 11rem;
+    height: 11rem;
+
+}
+
+.card__image-container>img {
+    border-radius: px4px 4px 0 0;
+}
+
+.card-element__main-text {
+    margin-left: 1rem;
+
+}
+
+
+/* card element bottom */
+.card-element__bottom {
+    display: flex;
+    justify-content: space-evenly;
+    border-top: var(--6C757D) solid 1px;
+}
+
+
+.card-button {
+    background-color: inherit;
+    padding: 1rem;
+    margin: 1rem;
+    font-size: 1.6rem;
+    border: none;
+    border-radius: 10%;
+}
+/* vertical-card */
+.card-vertical {
+    background-color: var(--PURE-WHITE);
+    width: 35rem;
+}
+
+.card-vertical-upper-text {
+    margin-left: 1rem;
+    margin-top: .5rem;
+}
+
+.card-vertical-img-cont {
+    width: 35rem;
+    height: 19.6;
```

Comment:
19.6rem or is it just unit less?

Reply:
Checked it man it was a big mistake by me resolved it Thanks again